### PR TITLE
updated instructions

### PR DESCRIPTION
**Pull request by cyruzin**
I had some problems following the old instructions, some errors, so I made an update.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,25 +48,42 @@ Docker
 
 In order to run own copy of the book with gitook within local container:
 
-1. Build container image
+1. Enable Docker experimental features with vim or other text editor
```
Comment (initBasti) [inline README.md]: s/or other text editor/or another text editor/

File: README.md [hunk 2]
```diff
@@ -48,25 +48,42 @@ Docker
...
        -p 4000:4000 \
        --name linux-insides-book \
        --hostname linux-insides-book \
-       localhost/linux-insides-book
+       linux-insides-book
    ```
-1. Open your local copy of linux insides book under this url
+
+4. Open your local copy of linux insides book under this url
    http://127.0.0.1:4000
```
Comment (initBasti) [inline README.md]: This wasn't true for me, I had to explicitly use: `http://localhost:4000`
```bash
basti@basti-TUXEDO-Book-XA1510:~$ hostname -i
127.0.1.1
```
So, I would say we should use `http://localhost:4000`.

File: README.md [hunk 3]
```diff
@@ -48,25 +48,42 @@ Docker
 
 In order to run own copy of the book with gitook within local container:
```
Comment (initBasti) [inline README.md]: While we are at it, let us fix those typos quickly:
s/run own copy/run your own copy/
s/gitook/gitbook/
s/within local container/within a local container/

**Review by initBasti — CHANGES_REQUESTED** [review]
I have tested your instructions and they work well, but I have found a few minor changes, that I would like to see fixed before merging this.
Thank you for your contribution!

**Comment by cyruzin** [comment]
> I have tested your instructions and they work well, but I have found a few minor problems, that I would like to see fixed before merging this.
> Thank you for your contribution!

Sure, I'll update the PR.

**Comment by cyruzin** [comment]
@initBasti done :)

**Pull request merged**